server/game/cards/03_TWI/events/UnmaskingTheConspiracy.ts

```diff
@@ -0,0 +1,37 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { WildcardCardType } from '../../../core/Constants';
+
+export default class UnmaskingTheConspirancy extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '0959549331',
+            internalName: 'unmasking-the-conspiracy',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Discard a card from your hand',
+            immediateEffect: AbilityHelper.immediateEffects.discardCardsFromOwnHand((context) => ({
+                cardTypeFilter: WildcardCardType.Any,
```

Comment:
I think this might actually cause problems if it's here, it's the default so we can remove it